code/game/jobs/job/engineering.dm

```diff
@@ -134,3 +134,31 @@
 			return FALSE
 		H.species.equip_survival_gear(H,1)
 		return TRUE
+
+/datum/job/janitor
+	title = "Janitor"
+	flag = JANITOR
+	department = "Engineering"
+	department_flag = ENGSEC
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the chief engineer"
+	selection_color = "#fff5cc"
+	access = list(access_janitor, access_maint_tunnels, access_engine, access_research, access_sec_doors, access_medical)
+	minimal_access = list(access_janitor, access_maint_tunnels, access_engine, access_research, access_sec_doors, access_medical)
+
+
+	equip(var/mob/living/carbon/human/H)

```

Comment:
Please change this to use the backpack type vars that the rest of this file uses.

Reply:
rraggfraggin' lohikar not updating civilian.dm